apps/admin-web/src/app/components/RouteItem.tsx

```diff
@@ -49,52 +49,63 @@ const RouteItem = ({ route, selectedItem, setSelectedItem }: Props) => {
 
   const _renderRouteTitle = () => {
     return (
-      <Typography variant="body1" sx={{ p: 2 }} style={{ color: 'black' }}>
-        {route.path}
-      </Typography>
+      <Box
+        sx={{
+          display: 'flex',
+          flexDirection: 'row',
+          overflow: 'hidden',
+          p: 2,
+          gap: 2,
+        }}
+      >
+        <Typography noWrap variant="body1">
+          <b>Path: </b>
+          {route.path}
+        </Typography>
+        {selectedItem !== route.id && activeVariant !== 'default' && (
+          <Typography noWrap variant="body1">
+            <b>Variant: </b>
+            {activeVariant}
+          </Typography>
+        )}
+      </Box>
     );
   };
 
   const _renderInteractiveButtons = () => {
     return (
       <Box sx={{ display: 'flex', flexDirection: 'row' }}>
-        {/* I think this is too much, I thought it was a button and was confused when it didn't work, accordion is enough */}
-        {/* {selectedItem === route.id && (
-          <Visibility
-            color="primary"
-            sx={{ alignSelf: 'center', mr: 1 }}
-          ></Visibility>
-        )} */}
-        <Button
-          variant="contained"
-          size="small"
-          sx={{
-            mr: 2,
-            backgroundColor: getColors().textColor,
-            maxHeight: 30,
-            alignSelf: 'center',
-          }}
-          onClick={(event) => {
-            event.stopPropagation();
-            if (route.method?.toUpperCase() === 'GET') {
-              openInNewTab(route.path.toString());
-            } else {
-              // If not a POST, currently assumes response is always JSON, fetch via API then open
-              fetch(route.path.toString(), {
-                method: route.method,
-                headers: {
-                  Accept: 'application/json',
-                  'Content-Type': 'application/json',
-                },
-                body: route.method === 'GET' ? undefined : '{}',
-              })
-                .then((r) => r.json())
-                .then(openJsonInNewTab);
-            }
-          }}
-        >
-          <OpenInNew />
-        </Button>
+        {selectedItem === route.id && (
+          <Button
+            variant="contained"
+            size="small"
+            sx={{
+              mr: 2,
+              backgroundColor: getColors().textColor,
+              maxHeight: 30,
+              alignSelf: 'center',
+            }}
+            onClick={() => {
+              if (route.method?.toUpperCase() === 'GET') {
```

Comment:
Route method will always be uppercase right?

Reply:
This change was implemented in a previous PR. However, this does guarantee that even if when a route is generated, it will, in this conditional always be uppercase even if by default it should be.